www/_posts/2024/2024-03-03-pyladies-at-pycon.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post.html
+title: "PyLadies presenting at PyCon US 2024"
+tags: [PyLadies, Global, Community]
+author: Naty Clementi
+author_link: https://twitter.com/ncclementi
+---
+
```

Comment:
Can you also add other PyLadies events:
- PyLadies booth at Expo Hall. Mention that we sell t shirts and you can get stickers
- PyLadies Luncheon. Mention that it is free but must sign up
- PyLadies Auction. Mention that registration is required. Also encourage people to donate items. Add link to the auction page.

Reply:
Sure thing, do you want this at the top or at the bottom as Extra events? 

For the "PyLadies Luncheon. Mention that it is free but must sign up" do we have a signup link yet? I didn't see when I registered

Reply:
Hmm I think it's not yet available.

Reply:
D

> Sure thing, do you want this at the top or at the bottom as Extra events?
> 
 At the bottom Is ok. Perhaps as "Events" instead of "Extra events"